src/desktop/lib/webpackPublicPath.ts

```diff
@@ -29,6 +29,8 @@ if (process.env.NODE_ENV === "production") {
     "/collections",
     "/collection",
     "/collect",
+    "/order",
```

Comment:
Note that this can likely be removed as its not used, but I wanted to match the [regex here](https://github.com/artsy/force/blob/add8b5f3a4d37ef6483ac80c2735083f2442bb07/src/v2/Apps/Order/orderRoutes.tsx#L29) for the time being.